packages/block-library/src/search/edit.js

```diff
@@ -303,6 +312,19 @@ export default function SearchEdit( {
 		</>
 	);
 
+	const getWrapperStyles = () => {
+		if ( 'button-inside' !== buttonPosition || ! style?.border?.radius ) {
```

Comment:
wouldn't the negated condition be easier?

```js
if('button-isnide' === buttonPosition && style.?border.?radius ) {
  // We have button inside wrapper and a border radius value to apply.
  // Add default padding so we don't get "fat" corners.
  const outerRadius =
    parseInt( style?.border?.radius, 10 ) + DEFAULT_INNER_PADDING;

  return { borderRadius: `${ outerRadius }px` };
}


```

Reply:
Thanks, I've cleaned this up 👍